lib/hypervisor/hv_kvm/__init__.py

```diff
@@ -47,10 +47,10 @@
   import psutil   # pylint: disable=F0401
   if psutil.version_info < (2, 0, 0):
     # The psutil version seems too old, we ignore it
-    psutil_err = "too old (2.x.x needed, %s found)" % psutil.__version__
+    psutil_err = "too old (2.x.x or newer needed, %s found)" % psutil.__version__
     psutil = None
-  elif psutil.version_info >= (3,):
-    psutil_err = "too new (2.x.x needed, %s found)" % psutil.__version__
+  elif psutil.version_info >= (6,):
```

Comment:
AFAICT, psutil does not follow semantic versioning rules (see [changelog](https://github.com/giampaolo/psutil/blob/master/HISTORY.rst)), so I think it's more futureproof to remove the upper version bound, than not do so :)

Reply:
I wonder if we can relax/change this so that we check if psutil understands what we want to do, rather than base check on version?

Reply:
@iustin Sorry, I force-pushed an update before reading your comment here.

I have no idea how this could be implemented, but I could look into it at some point if you think it makes sense. I don't think I'll be able to before 2.16.1 goes GA though...

Reply:
@gedia - oh sure, I didn't mean for this patch, but as general concept. The way it is right now looks good.